### PR TITLE
Release 2019.04.1

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **Release 2019.04.1**

### CODE REVIEW

File: packages/@okta/vuepress-site/.vuepress/nav/main.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ module.exports = [
           { title: 'Apps', link: '/docs/api/resources/apps/'},
           { title: 'Authorization Servers', link: '/docs/api/resources/authorization-servers/'},
           { title: 'Dynamic Client Registration', link: '/docs/api/resources/oauth-clients/'},
+          { title: 'Event Hooks', link: '/docs/api/resources/event-hooks/'},
```
Comment (motikagan-okta) [inline packages/@okta/vuepress-site/.vuepress/nav/main.js]: @bretterer, could you confirm whether this file should part of this pull request? I don't recognize this file.

File: packages/@okta/vuepress-site/docs/change-log/2019-04-1/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+title: Okta API Products Change Log
+---
+
+## 2019.04.1
```
Comment (motikagan-okta) [inline packages/@okta/vuepress-site/docs/change-log/2019-04-1/index.md]: When I try to build the web site from this branch locally on my machine, I don't see the new change log entry. I'm not sure what's going on. @bretterer, do you have time to take a look? This is what I see:
Screen Shot 2019-04-19 at 10 43 40 AM

File: packages/@okta/vuepress-site/.vuepress/nav/main.js (hunk 1 above)
Comment (bretterer) [reply packages/@okta/vuepress-site/.vuepress/nav/main.js]: If I remember correctly, it was added via one of the PR's that was merged into this release

File: packages/@okta/vuepress-site/docs/change-log/2019-04-1/index.md (hunk 2 above)
Comment (bretterer) [reply packages/@okta/vuepress-site/docs/change-log/2019-04-1/index.md]: Looks fine to me.

Screen Shot 2019-04-19 at 11 01 13 AM

Please make sure you are not cached and have a current version of this branch

Comment (motikagan-okta) [reply packages/@okta/vuepress-site/docs/change-log/2019-04-1/index.md]: Thanks for checking

Comment (motikagan-okta) [reply packages/@okta/vuepress-site/docs/change-log/2019-04-1/index.md]: Seems I had to do `yarn install`, now seeing the page correctly.

**Pull request merged**